### PR TITLE
Adding roadmap - editing readme

### PR DESCRIPTION
**Pull request by theJasonHelmick**
@sdwheeler / @dsajanice - Can you review?

@sdwheeler - pay extra attention to the readme. I corrected and reflowed, but I think your keen eyes will catch what I missed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,52 +1,77 @@
...
-
-When you connect to Azure Cloud Shell, we start a container containing a wide variety of tools, and connect your
-browser to a shell process running inside that container. This repository contains the Docker files used to build that image. 
-It does _not_ contain all of the code used for the rest of the Azure Cloud Shell service. The code in this repository may not 
-match exactly to what is running in the Cloud Shell service at any given time. The service is updated periodically and changes 
-are gradually rolled out to different regions over time, so there may be a lag of up to 3-4 weeks between a change being made 
-here and being reflected in all Cloud Shell regions.
+Azure Cloud Shell is an interactive, authenticated, browser-accessible shell for managing Azure
+resources. As a web-based environment, Cloud Shell provides immediate management access to any
+device with a network connection. Azure Cloud Shell provides a
```
Comment (sdwheeler) [inline README.md:6]: "any device"? Should that be "any Azure connected device"?

File: roadmap/cs-roadmap-2023.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+# Roadmap 2023 and future investigations
+
+Azure Cloud Shell is an interactive, authenticated, browser-accessible shell for managing Azure
+resources. As a web-based environment, Cloud Shell provides immediate management access to any
+device with a network connection and provides a consistent and secure environment with
```
Comment (sdwheeler) [inline roadmap/cs-roadmap-2023.md:5]: "any device"?

File: roadmap/cs-roadmap-2023.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+always-up-to-date tools. Users can work with their resources from any device, interactively, or
+through automation
+
+## FY2023 roadmap
+
+Azure Cloud Shell currently plans to release the following features and updates in the Sept-Oct 2023
+timeframe.
+
+We plan to focus and achieve these goals:
```
Comment (sdwheeler) [inline roadmap/cs-roadmap-2023.md:14]:
```suggestion
The Azure Cloud Shell team plans to release the following features and updates in the Sept-Oct 2023
timeframe.
```

File: roadmap/cs-roadmap-2023.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+### Improved User Interface
+
+The user interface refresh will help improve the customer experience, accessibility, and
+functionality of Cloud Shell. This will make it easier for users to navigate and find the tools and
+resources they need. Benefits include:
+
+- Improved user interface to be more modern and intuitive.
+- Accessibility improvements for users who may have disabilities or impairments by improving color
+  contrast, adjusting font sizes and other accessibility improvements.
```
Comment (sdwheeler) [inline roadmap/cs-roadmap-2023.md:50]: What "other accessibility improvements"?
```suggestion
- Update color contrast and font sizes to improve accessibility for user that may have vision impairments.
```

File: roadmap/cs-roadmap-2023.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+- Support for enhanced functionality
```
Comment (sdwheeler) [inline roadmap/cs-roadmap-2023.md:51]: Like what?

File: roadmap/cs-roadmap-2023.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+### Ephemeral Sessions
+
+Ephemeral sessions allow users to manage azure resources faster without the overhead of creating and
```
Comment (sdwheeler) [inline roadmap/cs-roadmap-2023.md:55]:
```suggestion
Ephemeral sessions allow users to manage Azure resources faster without the overhead of creating and
```

File: roadmap/cs-roadmap-2023.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+associating Azure storage accounts. These temporary and disposable sessions are best for scenarios
+where customers need to perform ad-hoc tasks without the need to persist and data or files between
```
Comment (sdwheeler) [inline roadmap/cs-roadmap-2023.md:57]:
```suggestion
where customers need to perform ad-hoc tasks without the need to persist data or files between
```

File: roadmap/cs-roadmap-2023.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+sessions. In addition, this provides a layer of security to scenarios to ensure that data or files
+created during the session are not accessible to other users or processes after the session ends.
+
+Benefits include:
+
+- Allow customers to leverage Cloud Shell without a linked storage account
+- Any saved files will be deleted with the container session
+- Can be leveraged by partners to ensure that a customer can obtain a Cloud Shell session even if a
+  storage account has never been configured
+- Benefits command injection scenarios
```
Comment (sdwheeler) [inline roadmap/cs-roadmap-2023.md:67]: Makes it easier to inject commands? What do you really mean?
```suggestion
- Benefits command injection scenarios
```

File: roadmap/cs-roadmap-2023.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+
+The VNET refresh will address these issues by making improvements:
+
+- Updating documentation to better guide customers through the process of enabling a VNET.
+- Improving the configuration options of the VNET scenario.
+- Providing guided assistance to customers through ready-made ARM templates.
+
+## FY2024 Investments, Investigations and discussions
+
+Working with customers, we are investigating and discussing future areas of imporvmeent and new features.  These investigation are greatly impacted and enhanced with customer feedback. We welcome your feedback as we progress through these discussions by adding issues or discussions to our repository.
```
Comment (sdwheeler) [inline roadmap/cs-roadmap-2023.md:84]:
```suggestion
We are working with customers to investigate and discuss areas of improvement and new features.
We welcome your feedback. Please add issues or discussions to our repository.
```

File: roadmap/cs-roadmap-2023.md [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+
+- Multiple saved storage accounts
+- Multiple/Custom images
+- Improved editor experience
+- Tenent/Subscription Administrative controls to manage Cloud Shell access.
+- Toolbar tabs
+- Improvements to Cloud Shell timeout values
```
Comment (sdwheeler) [inline roadmap/cs-roadmap-2023.md:91]: What is this list? It seems disconnected from the previous paragraph.

```suggestion
- Multiple saved storage accounts
- Multiple/Custom images
- Improved editor experience
- Tenent/Subscription Administrative controls to manage Cloud Shell access.
- Toolbar tabs
- Improvements to Cloud Shell timeout values
```

**Review by sdwheeler — CHANGES_REQUESTED** [review]
Several questions and many suggested changes.

File: README.md (hunk 1 above)
Comment (theJasonHelmick) [reply README.md:6]: Yes it should :)

File: roadmap/cs-roadmap-2023.md (hunk 10 above)
Comment (theJasonHelmick) [reply roadmap/cs-roadmap-2023.md:89]: Agreed -- added context

File: roadmap/cs-roadmap-2023.md (hunk 8 above)
Comment (theJasonHelmick) [reply roadmap/cs-roadmap-2023.md:64]: Great catch - this really doesn't make sense - and I don't think improving helps communicate the message for ephemeral -- so im removing

**Review by sdwheeler — CHANGES_REQUESTED** [review]
Remove the .DS_Store file. This is a metadata file that macOS adds. This should be added to the .gitignore file so that they won't keep showing up in the repo.

**Comment by theJasonHelmick** [comment]
> Remove the .DS_Store file. This is a metadata file that macOS adds. This should be added to the .gitignore file so that they won't keep showing up in the repo.

Fixed -- thank you!

**Comment by dsajanice** [comment]
Looks great! Thank you @theJasonHelmick for the content and @sdwheeler for the fine tuning!

**Review by dsajanice — APPROVED** [review]

**Pull request merged**